src/common/common-properties/build.gradle.kts

```diff
@@ -6,7 +6,7 @@ plugins {
 
 dependencies {
   implementation(libs.slf4j.api)
-  implementation(libs.microprofile.config.api)
+  implementation(libs.smallrye.config.core)
```

Comment:
that might introduce jboss logmanager into the classpath. Are there any errors in tests or similar?

Reply:
Needed for the 
@ConfigMapping(prefix = "xroad.common")
public interface CommonProperties { 
....